README.md

```diff
@@ -1,140 +1,158 @@
 
-Examples of what can be referenced using PAC-ID's across systems:
+_The Balance is known under **multiple** identifiers as each system issues its own._
 
-- substances, such as stock chemicals, reagents, standards or intermediates.
+As part of the Smart Building Blocks for Digital Labs initiative, the **publicly addressable content identifier ("PAC-ID")** addresses this by promoting the re-use of already existing identifiers. The identifier is augmented with information about the issuing system and is represented in standardized formats, thus making it globally unique and reusable across system borders.
 
-- instrument parameter or result sets, such as HPLC methods and results.
+![All systems are using the same globally unique identifier for referring to the balance.](images/balance-same.svg)
 
-- assets, such as lab devices.
+_All systems are using the **same** globally unique identifier for referring to the balance._
 
-- complete reports, such as analysis certificates or batch records, even among different companies, for example a service lab with its clients.
+The `PAC-ID` also serves as foundation for further Smart Building Blocks that provide functionality that wouldn’t be feasible with proprietary identifiers.
 
-## Example PAC-ID
+## Specification
 
-A PAC-ID can be represented in different ways, for example as a QR code. Such codes include an easy-to-recognize visual element on the side of the code:
+### Components
 
-![A PAC-ID](images/example-pac-id.png)
+A PAC-ID is composed of an `issuer` and `identifier` component. It is REQUIRED that the combination of `issuer` and `identifier` is globally unique.
 
-PAC-IDs can also be represented in textual form:
+| **Name** | **Meaning** | **Technical Requirements** | **Example** |
+| :--- | :--- | :--- | :--- |
+| `issuer` | The party which issued the identifier and knows what the identifier refers to. | <ul><li>MUST be a valid domain name according to [RFC 1035](https://www.ietf.org/rfc/rfc1035.html).</li><li>SHOULD be a registered and active domain name.</li><li>SHOULD contain only the following characters `A-Z`, `0-9`, `-`, and `.`</li></ul> | "METTORIUS.COM"<br>(The manufacturer of the balance) |
+| `identifier` | The identifier itself. | <ul><li>MUST consist of one or more `id segments` separated by `/`.</li><li>At least one `id segment` MUST be non-empty.</li><li>MUST not exceed 256 characters.</li></ul> | "DEVICE/21:210263"<br>(An identifier for one particular balance) |
+| `id segment` | The `id segment` is a part of an `identifier` that can stand on its own. Typically used to organize `identifier`s within an `issuer`. | <ul><li>MUST be a valid `hsegment` according to [RFC 1738](https://www.ietf.org/rfc/rfc1738.txt), but without `*` (see **PAC-ID Extension**). </li><li>SHOULD be limited to `A-Z`, `0-9`, and `:-+` for new designs.</li><li>CAN be an `id segment key` and `id segment value` pair separated by `:`.</li></ul> | "21:210263"<br>(A id segment containing a serial number) |
```

Comment:
typo in segment.

Reply:
Cannot find the typo @apinilabs-pascal . if you meant "hsemgent", than this is correct, it refers to RFC 1738. If you meant something else, please correct directly in V1.0 branch.